_pages/install/nightlies.md

```diff
@@ -3,134 +3,31 @@ page_title: Nightly Builds
-      <div class="install-entry">
-        <span class="title">Homebrew/<wbr />Linuxbrew</span>
-        {% highlight shell %}brew install crystal --HEAD{% endhighlight %}
-        <a href="https://formulae.brew.sh/formula/crystal" title="Crystal package on Homebrew" class="info">{% include icons/info.svg %}</a>
-      </div>
-      <div class="install-entry">
-        <span class="title"><a href="../from_snapcraft/">Snapcraft</a></span>
-        {% highlight shell %}snap install crystal --classic --edge{% endhighlight %}
-        <a href="https://snapcraft.io/crystal" title="Crystal on Snapcraft" class="info">{% include icons/info.svg %}</a>
-      </div>
-    </div>
-  </div>
-</div>
+{% include pages/install/section.html os="Linux" channel="nightly" %}
 
 ## MacOS
 
-<div class="install-panels">
-  <div class="install-group">
-    <h5>Crystal</h5>
-    <div class="install-entries">
-      <div class="install-entry">
-      <span class="title"><a href="../from_targz">Tarball</a></span>
-        <p>Archive on <a href="https://artifacts.crystal-lang.org/dist/crystal-nightly-darwin-universal.tar.gz">artifacts.crystal-lang.org</a>.</p>
-      </div>
-    </div>
-  </div>
-  <div class="install-group">
-    <h5>Community</h5>
-    <div class="install-entries">
-      <div class="install-entry">
-        <span class="title">Homebrew/<wbr />Linuxbrew</span>
-        {% highlight shell %}brew install crystal --HEAD{% endhighlight %}
-        <a href="https://formulae.brew.sh/formula/crystal" title="Crystal package on Homebrew" class="info">{% include icons/info.svg %}</a>
-      </div>
-    </div>
-  </div>
-</div>
+{% include pages/install/section.html os="MacOS" channel="nightly" %}
 
 <a id="windows"></a>
 
 ## Windows (preview)
 
-<div class="install-panels">
-  <div class="install-group">
-  <h5>Crystal</h5>
-    <div class="install-entries">
-      <div class="install-entry">
-        <span class="title"><a href="../on_windows">Portable Archive</a></span>
-        <p>
-          Archive on <a href="https://nightly.link/crystal-lang/crystal/workflows/win/master/crystal.zip">artifacts.crystal-lang.org</a>.
-        </p>
-      </div>
-    </div>
-  </div>
-  <div class="install-group">
-    <h5>Community</h5>
-    <div class="install-entries">
-      <div class="install-entry">
-        <span class="title"><a href="../from_scoop">Scoop</a></span>
-        {% highlight powershell %}
-scoop install git
-scoop bucket add crystal-preview https://github.com/neatorobito/scoop-crystal
-scoop install crystal-nightly{% endhighlight %}
-        <a href="https://github.com/neatorobito/scoop-crystal" title="Scoop repository for Crystal on GitHub" class="info">{% include icons/info.svg %}</a>
-      </div>
-    </div>
-  </div>
-</div>
+{% include pages/install/section.html os="Windows" channel="nightly" %}
 
 ## Docker
 
 Nightly builds are available on the `nightly` tag on the [Docker repository of Crystal](https://hub.docker.com/r/crystallang/crystal/).
```

Comment:
This is now kind of redundant, isn't it?

Reply:
Yes, it'll be removed in a follow-up.